pages/common/gulp.md

```diff
@@ -0,0 +1,11 @@
+# gulp
+
+> JavaScript task runner and streaming build system.
```

Comment:
It so happens I started using Gulp for the first time yesterday. To slightly touch @waldyrious's comment, I'd say we compromise and have a line here saying that tasks are defined in a Gulpfile. That way we don't really have to explain what tasks are. Whaddayathink @waldyrious?